catbot.py

```diff
@@ -0,0 +1,106 @@
+"""
+
+Demo bot: catbot.
```

Comment:
Maybe we can add a message here to explain what it's good for:
```suggestion
Demo bot: catbot.

This bot uses all options provided by the Poe protocol. You can use it to get examples
of all the protocol has to offer.
```